package.json

```diff
@@ -3,14 +3,14 @@
   "scripts": {
     "bootstrap": "lerna bootstrap",
     "clean": "lerna clean",
-    "updated": "lerna updated",
     "diff": "lerna diff",
-    "publish": "lerna publish",
-    "dev": "lerna publish --npm-tag=dev --skip-git",
-    "rc": "lerna publish --npm-tag=rc --skip-git",
-    "force": "lerna publish --force-publish=*"
+    "updated": "lerna updated",
+    "release": "lerna publish",
```

Comment:
So a release will be `lerna publish --yes`

How does that resolve versions?

e.g. 

```
a@1.1.0

b@2.0.0 depends on a@^1.1.0

c@3.0.0 depends on a@^1.1.0

d@4.0.0
```

If you merged changes to master that constitute a minor bump of:

`a@1.1.0` to `a@1.2.0`
  - causes `b@2.0.0` to become `b@2.0.1`
  - causes `c@3.0.0` to become `c@3.0.1`

and a major of:

`d@4.0.0` to `d@5.0.0`

as well as a small change of:

`c@3.0.0` to `c@3.0.1`

How would that all get resolved non interactively?

The result needs to be:

```
a@1.2.0
b@2.0.1
c@3.0.1
d@5.0.0
```